TravelAgencyService/Models/AgencyModel.cs

```diff
@@ -0,0 +1,63 @@
+namespace TravelAgencyService;
+
+public class AgencyModel
+{
+    private readonly List<DestinationProcessingService.Destination> _destinations;
+
+    public AgencyModel()
+    {
+        _destinations = new List<DestinationProcessingService.Destination>
```

Comment:
You can set this values on field level, without constructor. It will work absolutely the same